xclim/indicators/atmos/_precip.py

```diff
@@ -307,7 +316,7 @@ class HrPrecip(Hourly):
 )
 
 
-fraction_over_precip_thresh = Precip(
+fraction_over_precip_doy_thresh = Precip(
     identifier="fraction_over_precip_thresh",
```

Comment:
This identifier should be different than the one of `fraction_over_precip_thresh`.